spec/controllers/students_controller_spec.rb

```diff
@@ -541,4 +541,50 @@ def make_request(query)
     end
   end
 
+  describe '#sped_referral' do
+    let(:educator) { FactoryGirl.create(:educator, :admin, school: school) }
+    let(:school) { FactoryGirl.create(:school) }
+    let(:service) { FactoryGirl.create(:service, student: student) }
+    let(:student) { FactoryGirl.create(:student, :with_risk_level, school: school) }
+    let(:student_school_year) { FactoryGirl.create(:student_school_year, student: student) }
+
+    def make_request(options = { student_id: nil, format: :pdf })
```

Comment:
So in all these tests, we're making requests to the endpoint, then checking that the response contains something, right?